app/src/main/java/org/wikipedia/suggestededits/SuggestedEditsTasksFragment.kt

```diff
@@ -98,6 +99,12 @@ class SuggestedEditsTasksFragment : Fragment() {
         clearContents()
     }
 
+    private fun Group.addOnClickListener(listener: (view: View) -> Unit) {
+        referencedIds.forEach { id ->
+            rootView.findViewById<View>(id).setOnClickListener(listener)
+        }
+    }
```

Comment:
Would it be better if set the parameter as `listener: View.OnClickListener`

It would be much clear when adding listeners to it.
```
userStatsViewsGroup.addOnClickListener(View.OnClickListener {
            startActivity(ContributionsActivity.newIntent(requireActivity()))
})
```